config.json

```diff
@@ -0,0 +1,108 @@
+{
+	"version": "16",
```

Comment:
We have moved to config version 18 the latest release.. perhaps this should reflect that.